certif/tests/helpers/extended-ember-test-helpers/click-by-label.js

```diff
@@ -2,6 +2,7 @@ import { getScreen } from '@pix/ember-testing-library';
 import { click } from '@ember/test-helpers';
 
 export default function clickByLabel(labelText) {
+  // eslint-disable-next-line no-console
   console.warn(`The clickByLabel helper is deprecated. Use testing-library's click(getByRole('${labelText}')) instead.`);
```

Comment:
Pas de rapport avec cette PR mais du coup je m'aperçois d'une erreur dans le message :

```suggestion
  console.warn(`The clickByLabel helper is deprecated. Use testing-library's click(getByRole(role, { name: '${labelText}' })) instead.`);
```